src/databaseInterface/src/main/java/org/vaccineimpact/api/db/annex/Keys.java

```diff
@@ -55,6 +55,6 @@
 
     private static class UniqueKeys0 {
         public static final UniqueKey<BurdenEstimateStochasticRecord> BURDEN_ESTIMATE_STOCHASTIC_PKEY = Internal.createUniqueKey(BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC, "burden_estimate_stochastic_pkey", BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.ID);
-        public static final UniqueKey<BurdenEstimateStochasticRecord> BURDEN_ESTIMATE_STOCHASTIC_UNIQUE = Internal.createUniqueKey(BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC, "burden_estimate_stochastic_unique", BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.BURDEN_ESTIMATE_SET, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.MODEL_RUN, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.COUNTRY, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.YEAR, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.AGE, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.BURDEN_OUTCOME);
+        public static final UniqueKey<BurdenEstimateStochasticRecord> BURDEN_ESTIMATE_STOCHASTIC_UNIQUE = Internal.createUniqueKey(BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC, "burden_estimate_stochastic_unique", BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.BURDEN_ESTIMATE_SET, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.COUNTRY, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.YEAR, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.AGE, BurdenEstimateStochastic.BURDEN_ESTIMATE_STOCHASTIC.BURDEN_OUTCOME);
```

Comment:
We should think about the ordering on the unique constraint - although JOOQ may be getting it wrong, so the migration is the thing to examine